### ISSUE TITLE
Feedback & Suggestion 

### ISSUE THREAD

**Issue by Siddhant-K-code** [Marvel999/Time-Management-App-DOIT-#3]
Nice Work Buddy ! 

Suggestions : -

* i think there is no need of log in & sign up .
* you can also add category of to do work.
* you can add  direct download link of your APP

**Comment by Marvel999** [comment]
yes, I will definitely do that thank you for your suggestions and feedback.
